cics/CICSCBL.OPT

```diff
@@ -0,0 +1,6 @@
+  cics
+  optable(Z390)
+  stats
+  syscpy(cics+zcobol/CPY+zcobol/MAC+MAC+.)
+  sysmac(cics+zcobol/MAC+MAC)
```

Comment:
Change MAC to mac. Four occurrences. Left as is will break bash scripts.

This appears many times in these OPT files. I will leave it up to you to find and change them.